src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -182,10 +182,16 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
 
         # Run agent parameters
         c.argument('cpu', type=int, help='The CPU configuration in terms of number of cores required for the run.')
+        
+        # MSI parameter
+        c.argument('assign_identity', nargs='*', help="Assigns a Managed Identity to the task. Use '[system]' to refer system assigned identity, or a resource id to refer user assigned identity.")
```

Comment:
why is a parameter named as a verb? ignore this comment if you are following existing patterns from other services.

Reply:
This is an existing pattern